homeassistant/components/tesla/sensor.py

```diff
@@ -78,7 +78,7 @@ def unit_of_measurement(self):
                 self._unit = TEMP_FAHRENHEIT
             else:
                 self._unit = TEMP_CELSIUS
-        elif self.tesla_device.bin_type == 0xA or self.tesla_device.bin_type == 0xB:
+        elif self.tesla_device.type in ["range sensor", "mileage sensor"]:
             self.current_value = self.tesla_device.get_value()
             tesla_dist_unit = self.tesla_device.measurement
```

Comment:
We already have `units` available as a variable.

Reply:
Ok, changing `_unit` to `units`.